client/src/components/PosterList.spec.js

```diff
@@ -0,0 +1,31 @@
+import { render, screen } from '@testing-library/react'
+import PosterList from './PosterList'
+
+const list = [
+  { id: 1, alt: 'Poster of Title 1' },
+  { id: 2, alt: 'Poster of Title 2' },
+  { id: 3, alt: 'Poster of Title 3' },
+]
+
+describe('PostList', () => {
```

Comment:
You don't test the `alt` attributes. You could add a test for that with `getByAltText`.